### ISSUE TITLE
Enable searching or displaying which categories/projects another user can access

### ISSUE THREAD

**Issue by january3** [bihealth/sodar-core#1498]
### Problem

Quite often, it is required to find a project of a particular user who does not remember what the project ID or the project name was. On other occasions one would like to know what projects a specific user has access to (which I, the other user, have access to as well).

### Solution

Keywords entered into the search box should be also compared against user first and last names as well as their usernames.

**Comment by mikkonie** [comment]
I agree that there should be a way to display all categories/projects in which a user has access. This would be useful for many use cases.

However, I'm not sure if this should happen through the search box. Another option would be to allow users to see (the public part of) other users' user profiles, where a list of categories/projects both of them have access to would also be displayed. That is related to #425, although that issue only deals with superusers specifically.

Regardless of the implementation, the results should only lists projects both the querying user and the subject user can access.

I'm open for comments and ideas on what would be the best way to achieve this. I do agree it is information that should be accessible.

**Comment by mikkonie** [comment]
This will be handled in #1855. Closing this issue as a dupe.
